ui/src/submissions/seminars/schemas/seminar.js

```diff
@@ -15,7 +16,7 @@ const seminarSchema = object().shape({
     .required()
     .label('Seminar Name'),
   dates: array()
-    .of(date())
+    .of(date(SEMINAR_DATETIME_FORMAT))
```

Comment:
can we tests this?

Reply:
it's tested but I can add failure case for dates of different format, which won't happen because UI component is enforcing the format.